flink-autoscaler-plugin-jdbc/src/test/java/org/apache/flink/autoscaler/jdbc/event/AbstractJdbcAutoscalerEventHandlerITCase.java

```diff
@@ -459,7 +462,8 @@ void testDeleteCounterWhenIdNotConsecutive() throws Exception {
                         .max(Comparable::compareTo)
                         .orElseThrow();
 
-        try (Connection connection = getConnection();
+        var dataSource = getDataSource();
```

Comment:
This temporary data source isn't closed. 

Or could we reuse the dataSource of `beforeEach` instead of creating a new one?